### ISSUE TITLE
Provide AppImage for download on GitHub Releases

### ISSUE THREAD

**Issue by probonopd** [calamares/calamares#1114]
Now that there is

https://github.com/calamares/calamares/blob/427b09e915a003edc0a4b9a7a96b762c14b0a635/ci/AppImage.md

it would be cool if one could "just download" it for testing purposes.
Please provide an AppImage (built on Ubuntu trusty or xenial but no newer) for download on GitHub Releases.

**Comment by abucodonosor** [comment]
Why should we do such thing ? And why based on Ubuntu ? Why not ArchLinux ? or Suse ? or Debian ?

But even *when* Ubuntu does not support calamares. So what you really want is to ask Distros supporting
calamares to provide an appimage since just having calamares compiled in there does nothing much without a proper configuration , which again is based on the *Distribution* , and even now other than for
and OEM install it does not make any sense to use an appimage. ( and even there is a big question whatever it works .. ).

In other words *just having calamares in a appimage or similar* is useless and you cannot
*just test anything*

**Comment by probonopd** [comment]
> Why should we do such thing?

To make it easy to download and test it.

> And why based on Ubuntu? Why not ArchLinux? or Suse? or Debian?

Ubuntu may be interesting because is natively supported on [Travis CI](https://travis-ci.org/) which can be used for continuous builds. If done right, the resulting AppImage will run on ArchLinux, Suse, and Debian as well. But my point was more the "trusty or xenial but no newer", so that the AppImage will run on other but the very latest distributions.

**Comment by adriaandegroot** [comment]
Hi @probonopd .. Calamares can be built as an AppImage, sure (and I twitted about it, and talked with Alexis about it as well). It doesn't make much sense to make it (as in, the executable, plus maybe some standard configuration) available for end-user download. It's a system installer. It won't work unless .. well, unless you have a system to install with it. That means all the configuration bits, and a squashed filesystem to work with.

I suppose that having the image available would allow **system integrators** and **distribution maintainers** to try Calamares a little easier, and to allow some experimentation with a newer Calamares (in a live CD) before packaging it. I'm hesitant, though: distro's are **supposed** to be building Calamares + configuration themselves, carefully, for their specific distribution release. That's what it's for, and otherwise it can be a remarkably destructive tool.

**Comment by probonopd** [comment]
OK, I understand @adriaandegroot. Thanks for taking the time to explain.
